blog/models.py

```diff
@@ -68,6 +82,61 @@ def get_absolute_url(self) -> str:
         :rtype: str
         """
         return reverse('ministrant-detail', kwargs={'pk': self.pk})
+    
+    @property
+    def variable_symbol(self):
+        return f'{self.get_actual_camp_year()}{self.pk:04d}'
+
+    @property
+    def unicode_name(self) -> str:
+        return unidecode(f'{self.surname}_{self.birthname}')
+
+    def get_actual_camp_year(self) -> int:
+        return SummerCampInfo.objects.first().start_date.year
+
+    def generate_qr_pay_code(self):
```

Comment:
IMHO the QR code functionality could be written as a separate class, because it is not a model. This is not necessary of course, as this way it also works. But I think that separating the QR code logic from the models would lead to clearer code structure.

Reply:
Yes, I understand you. And when I started with QR pay code generator, I writed it as a separated class but it didnt work. So, for fuctionality I decided to write it first as a local function. But yes, it is good approvement make it separated in a class.